pqEventObserver.h

```diff
@@ -60,10 +60,16 @@ class QTTESTING_EXPORT pqEventObserver :
   virtual void setStream(QTextStream* stream);
 
 public slots:
+  // Slot called when recording an event
+  // Event Type can be pqEventTypes::EVENT or pqEventType::CHECK_EVENT
+  // Widget, Command/Property and Arguments are QString
+  // THis method has to be redefined by subclasses
   virtual void onRecordEvent(
+    const int& eventType,
```

Comment:
in other classes eventType is after widget and command.
Could there be a consensus ?